### DISCUSSION TITLE
[A-L] (2023/24) Foglio 9 - Esercizio 6

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#200] [Esercizi - Bertini]
![Screenshot 2023-11-27 134951](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/7fa8529b-8252-48a0-ac43-a5364c913f91)

**Comment by cornflexxx** [comment]
### Soluzione del Prof
$1)$ Calcolo la distribuzione di $V^2$ definita in un intervallo $[0,1]$:

$$\mathbb{P}\\{a\le V^2 \le b\\}=\mathbb{P}\\{\sqrt{a}\le V \le \sqrt{b}\\}+\mathbb{P}\\{-\sqrt{a}\le V \le -\sqrt{b}\\}=2\mathbb{P}\\{\sqrt{a}\le V \le \sqrt{b}\\} =\int^{\sqrt{b}}_{\sqrt{a}} 2f_v(v)dv=\cdots$$

$$\cdots=\begin{pmatrix}
x=v^2\\
v=\sqrt{x}\\
dx=2vdv\\
dv=\frac{1}{2\sqrt{x}}
\end{pmatrix}=\int_a^b \frac{1}{2\sqrt{x}} dx \longrightarrow f_{v^2}(x)=\frac{1}{2\sqrt{x}}$$

$2)$  analizziamo la funzione di ripartizione di $Y=log(\frac{1}{U})$:


$$F_y(y)=\mathbb{P}\\{Y\le y\\}=\mathbb{P}\\{log\frac{1}{U}\le y\\}=\mathbb{P}\\{\frac{1}{U}\le e^y\\}=\mathbb{P}\\{U\ge e^{-y}\\}=\int^1_{e^{-y}} f_u(u) du=1-e^{-y}$$

$$f_y(y)=\frac{d}{dy} F_y(y)=e^{-y}$$

**Reply by cornflexxx** [reply]
$3)$
 Abbiamo che per $y\le 1$:
$$F_U(y)=\mathbb{P}(U\le y)=\int^y_0 du=y$$
 $$F_V(y)=\mathbb{P}(V\le y)=\int^y_{-1} \frac{1}{2}dv=\frac{1}{2}y+\frac{1}{2}$$

Confrontando le due funzioni vediamo che $F_U(y)\le F_V(y)$  nell'intervallo $[0,1]$ , $F_U$ è sempre sotto $F_V$ e coincidono chiaramente nel punto $(1,1)$ quindi : 
$$\mathbb{P}(U\le V)=1$$

**Comment by Elia-Belli** [comment]
**(1)** Ricavo la funzione di distribuzione di $V^2$ e la derivo per calcolare la densità.

$$F_{V^2}(x)=\mathbb{P}(V^2\leq x)=\mathbb{P}(-\sqrt x \leq V \leq \sqrt x)=\mathbb{P}(V\leq \sqrt x)-\mathbb{P}(V\leq -\sqrt x)$$

Sappiamo che $V\sim Unif([-1,1])$ allora $f_V(x)=1/2$ e $F_V(x)=\frac{x}{2}$:

$$F_{V^2}(x)=\frac{\sqrt x}{2}- \frac{-\sqrt x}{2}=\sqrt x$$

$$f_{V^2}=\frac{d\ F_{V^2}}{dx}\to f_{V^2}(x)=\frac{1}{2\sqrt x}\ \ \ \ \ \ se\ 0\leq x\leq 1\ \ \ \ \ altrimenti\ 0$$

**Comment by CasuFrost** [comment]
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/97389762/86100739-f7c2-441b-9f8b-62cad3c7f222)
